lib/web_ui/lib/src/engine/html/recording_canvas.dart

```diff
@@ -1134,6 +1164,29 @@ class PaintDrawShadow extends DrawCommand {
   }
 }
 
+class PaintDrawPicture extends DrawCommand {
+  final EnginePicture picture;
+
+  PaintDrawPicture(this.picture);
+
+  @override
+  void apply(EngineCanvas canvas) {
+    final ui.Size windowSize = window.physicalSize;
+    final ui.Rect bounds =
+        ui.Rect.fromLTWH(0, 0, windowSize.width, windowSize.height);
```

Comment:
I don't think this is the right value for the bounds. The picture may be drawn while `canvas.scale` is in effect, which changes the drawable bounds of the picture. However, `window.physicalSize` is always the same. One way is to append picture draw commands to the parent recording canvas' command list, i.e. something like (pseudo-code):

```
void drawPicture(Picture picture) {
  // Need to save to make sure we don't pick up leftover clips and transforms from the picture
  save();
  for (DrawCommand childCommand in picture.recordingCanvas.commands) {
    _commands.add(childCommand);
  }
  restore();
}
```

We also won't need `PaintDrawPicture` command, as we'd be merging everything into the existing command list. For `save`/`restore` we probably need to remember the save count, because the picture may not restore everything.

Reply:
Done. The reason for PaintDrawPicture was opportunity to reuse/optimize pictures. Can be done in the future.

Reply:
Have you uploaded the latest iteration? I don't see the changes.

Reply:
You should see update now with _commands.addAll(pictureRecording._commands);